docs/user/password_reset.md

```diff
@@ -0,0 +1,45 @@
+# Password Reset
+
+The data manager application provides an easy and secure solution
+to set a new password should a password associated with a user account be lost.
+To reset your password the following steps have to be taken
+
+1. [Provide the account credentials for which the password should be reset](#trigger-password-reset)
+2. [Follow the password reset link you received in your email](#validate-the-password-reset-trigger)
+3. [Set a new password for the account according to the password policy](#set-a-new-password-for-the-account)
+
+## Trigger Password Reset
+
+From the login page you can navigate to a dedicated password reset site via the "Forgot password"
```

Comment:
```suggestion
From the login page you can navigate to a dedicated password reset dialog via the "Forgot password"
```

Reply:
Let's compromise on "view". A dialog is in our context a pop up window into which the user will provide his information

Reply:
agreed!